### DISCUSSION TITLE
Is there any documenation for server side filtering using P-table

### DISCUSSION THREAD

**Discussion by sulaimaan26** [primefaces/community#2656] [PrimeNG]
As of now i don't found any sources regarding this. Anyone please help

**Comment by JIBIN-P** [comment]
https://primeng.org/table#paginator-programmatic

You can take this as a reference and build your solution around it.

**Comment by qwadrox** [comment]
@sulaimaan26 There is no such documentation provided by PrimeTek, as the underlying functionality would depend on the backend implementation.
 
Try to find youtube videos such as: https://www.youtube.com/watch?v=iSd0w-wQeV0
and update the code in the video if needed based on the [API documentation](https://primeng.org/table/#api.table.events.TableFilterEvent)
